bookwyrm/templates/feed/layout.html

```diff
@@ -1,9 +1,26 @@
 {% extends 'layout.html' %}
 {% load i18n %}
+{% load utilities %}
 
 {% block title %}{% trans "Updates" %}{% endblock %}
 
 {% block content %}
+{% if user.moved_to %}
```

Comment:
Is it desirable, in the bigger picture, to have this panel only obscure the feed page? I would think it makes more sense for the whole interface to be blocked, but I realize that's a more complicated endeavor, and I also might be missing some reason to keep other views active.

If it would be better to block the entire UI, I think that could also fall outside the purview of this PR

Reply:
Happy to take advice on this. I think it's important that people have the ability to change their mind and not lose too much. Mastodon, for example, makes moves fairly catastrophic if you change your mind because it automatically triggers an unfollow for all your followers, and has fairly draconian cool-off periods.

On the other hand it needs to be clear that you shouldn't be doing anything with a moved account other than perhaps "unmoving" it.

To be honest I wasn't quite sure what to do for this aspect, and was feeling a bit overwhelmed by the various options. I tend to agree that the better option is to exclude the user from doing anything much other than changing their mind about the move. It's probably not much work, I guess I wasn't confident about making the call one way or the other on my own.